glossary.asciidoc

```diff
@@ -28,13 +28,13 @@ Balance::
     Nodes should aim to have balanced channels with similar inbound and outbound capicities.
 
 bech32::
-    A Base-32 + checksum bitcoin address format that is native to segregated witness. Also known as "bc1" address by starting letters. Specified in BIP-0173.
+    A checksummed base32 address format. It is native to segregated (BIP173). Also reffered to as "bc1" because of the starting values of each address. Transactions made using bech32 are smaller in most cases, and therefore, may only require a lower fee.
```

Comment:
```suggestion
    A checksummed base32 address format. It is native to segregated (BIP173). Also referred to as "bc1" because of the starting values of each address. Transactions made using bech32 are smaller in most cases, and therefore, may only require a lower fee.
```

Reply:
Thanks for catching that. What dev environment would you recommend? I'm planning on contributing a significant amount of time to this project and to lnd this and next year. 

I'm using goLand to get started with lnd.

Reply:
Thanks for catching that. What dev environment would you recommend? I'm planning on contributing a significant amount of time to this project and to lnd this and next year. 

I'm using goLand to get started with lnd.

Reply:
This was not resolved.